src/main/java/seedu/address/logic/CommandList.java

```diff
@@ -2,60 +2,86 @@
 
 import java.util.ArrayList;
 
+import seedu.address.logic.commands.NextCommand;
+import seedu.address.logic.commands.PreviousCommand;
+import seedu.address.logic.commands.exceptions.HistoryException;
+
+
 /**
  * Records the latest user inputs.
  * Only the latest MAX_RECORD_NUMBER inputs will be recorded.
  */
 public class CommandList {
-    private static final int MAX_RECORD_NUMBER = 3;
-    private static final ArrayList<String> inputHistory = new ArrayList<>();
+    private static CommandList theList = null;
+    private static final int MAX_RECORD_NUMBER = 10;
```

Comment:
Could you change this to 20 so that it's consistent with the `UNDO_REDO_CAPACITY` 😄